src/messages.js

```diff
@@ -21,6 +21,12 @@ Check POA Network <a href='https://github.com/poanetwork/wiki' target='blank'>wi
 messages.ballotIsNotActiveMsg = function(timeToStart) {
 	return `The ballot is not active yet. Time to start: ${timeToStart}`;
 };
+messages.SHOULD_BE_MORE_THAN_TWO_DAYS = (duration, neededHours, neededMinutes) => {
+	return `Ballot end time should be at least 48 hours from now in UTC time. Current duration is ${duration} hours
```

Comment:
Let's add dot after `Current duration is ${duration} hours`:

![screen shot 2018-01-22 at 16 23 47](https://user-images.githubusercontent.com/4341812/35222927-bc40a73c-ff90-11e7-97ce-8b7c988061db.png)